p2p_wallet/Scenes/Main/BankTransfer/KYC/KYCCoordinator.swift

```diff
@@ -60,6 +61,7 @@ final class KYCCoordinator: Coordinator<KYCCoordinatorResult> {
                 // TODO: handle BankTransferError.kycRejectedCantRetry and BankTransferError.kycAttemptLimitExceeded when more info is provided
                 notificationService.showDefaultErrorNotification()
                 await cancel()
+                await logAlertMessage(error: error, source: "striga api")
```

Comment:
there is BankTransferError.kycVerificationInProgress higher, should we handle it too?

Reply:
I think it is not an error to log.